pages/install/install.rst

```diff
@@ -14,149 +14,55 @@
 
    .. class:: lead
 
-      The following instructions detail how to install Cantera on a variety of platforms. We highly
-      recommend that all new users install the Python interface via Conda. If you want to use one of
-      the other interfaces, please select the installation instructions for your platform.
+      The following instructions detail how to install the different Cantera interfaces
+      on a variety of platforms. We highly recommend that all new users install the
+      Python interface via :ref:`Conda <sec-install-conda>`.
```

Comment:
Since users can also do the Matlab interface from Conda, now, would something like

> We highly recommend that all new users seeking a Python or Matlab interface install Cantera via :ref:`Conda <sec-install-conda>`.

work?

Reply:
Given the many shortcomings of the current Matlab toolbox, I would not recommend it on an equal footing with the Python module. This isn't just a question of what's easiest to install.